### ISSUE TITLE
The Use of the Name "Geometry Dash"

### ISSUE THREAD

**Issue by astraalunaa** [BrittonFischer/GeometryDash#9]
Sure this is some community driven game non profit whatever, but even so, I don't think using the name "Geometry Dash" in the game is a good idea.

"Geometry Dash" is a trademark to RobTop Games, and using their name could lead the game to be taken down

**Comment by nottirb** [comment]
Yeah I agree, I'll look to change that in the next release. I'll leave this issue open until then.

**Comment by 4812571** [comment]
I give you permission to use my "Geometric Rush" moniker
